Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM eclipse-temurin:11-jdk-focal
 
-ARG MAVEN_VERSION=3.8.7
+ARG MAVEN_VERSION=3.9.0
 # https://downloads.apache.org/maven/maven-3/3.8.7/binaries/apache-maven-3.8.7-bin.tar.gz.sha512
```

Comment:
Could this comment be updated or removed?

Reply:
Updated in https://github.com/jenkinsci/core-pr-tester/pull/29/commits/1601905dc74c1b353b0ac44dc8d74cda93113fee